packages/angular-cli/lib/config/schema.json

```diff
@@ -61,6 +61,12 @@
           "test": {
             "type": "string"
           },
+          "coverageExcludes": {
```

Comment:
Could you give me an example of code (outside of specs and node_modules) you want to exclude?

I think a smarter default to exclude everything outside of the appRoot as well would be better.